packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/quanticRefineModalContent.js

```diff
@@ -99,24 +112,37 @@ export default class QuanticRefineModalContent extends LightningElement {
     this.headless = getHeadlessBundle(this.engineId);
     this.searchStatus = this.headless.buildSearchStatus(engine);
     this.breadcrumbManager = this.headless.buildBreadcrumbManager(engine);
+    this.initializeSortOptions();
+
     this.unsubscribeSearchStatus = this.searchStatus.subscribe(() =>
-      this.gatherFacets()
+      this.initializeFacets()
```

Comment:
the previous name is more reflection to what the method and this component are actually doing.

Reply:
Hehe that's following one of my comment, perhaps initialize isn't the best word, but `gather` doesn't really mean anything either.

Not sure I 100% understand the implication of what this "store" is doing.

1. What is `this.data` supposed to be? I think perhaps now that we have `sortData` we should rename `this.data` to something like `facetData` ?
2. And then this method can become `initializeFacetFromStore` perhaps? or `getFacetDataFromStore` ?

Overall `gather` isn't really the right word to use here.

Reply:
im ok with `getSOMEDATAFromStore`, will rename the this.data as well